### ISSUE TITLE
[Bug Report][3.7.0] Can't open a v-list-group programmatically via its parent v-list's v-model:opened

### ISSUE THREAD

**Issue by piris-jose** [vuetifyjs/vuetify#20354]
### Environment
**Vuetify Version:** 3.7.0
**Vue Version:** 3.4.38
**Browsers:** Firefox 129.0
**OS:** Ubuntu undefined

### Steps to reproduce
Click the "Open Users" button.

### Expected Behavior
The Users v-list-group should be opened.

### Actual Behavior
The Users v-list-group doesn't get opened.

### Reproduction Link
[https://play.vuetifyjs.com/#...](https://play.vuetifyjs.com/#eNqdU82O2jAQfpVRLgGJwFa9RYC6D1DtoeoJOHgdA9Y6thVPKFWUd+/Y4PxsWA692J6Zb/6+8eyaxFV89Wrt8lKLJE/WKEqrGIrtXgOsLxlnVQFcMec2+6S8ZqxGs0/gjyzwTJrvLy/7JGAD+h01/OBK8g+yGSv0bycqRwh4IwGCBOtVAN69mgYa8FBooW27SEo6hEtWmkKo3JtFcQ/Z5etw2akytYULU7UgUMwZUYSLXVFEpwzmjKO8MDQVwRuwlbEO2pFLH1yS81APFORdal9OcNwnY6utBFVZZJIb7TkrZMY4N7XGjMuKK/HZASWqQd1D49Zz1VUx7Gg1GtQzOl6LUg5J+29CxpRMWOj6iAm/qv2L6p8zfgylzXYhyQI8uQe65iA1MJ9wMof8Q/wlFznRfxqQvyaY2Eu4J9ZI7QPrk4kNp9N/4UfqqLwtIQl+C0kY8Uai45W0N5S4WlMhFOLIaoXQ3AKVAs+mcHmUIWxa+Gizea8EkEeY4Vm6pbcvaa7i+nacpQGZzmGz2UD2beRB/7bD29qdO3APuW8zPRa3V7wLhiwHqmCzhdmotBx2hzsI7nMlVR9yl/5kmp1EKTSmC0iH21VS49IqkZkaldQi7SN5x18CUeqTi27cnB4g46udhwe1sF51NCftQtdKhePwD8ockF4=)

### Other comments
The v-model:opened is printed on the preview too. When you click the button, you can see that a 'Users' string is programmatically pushed into the v-model:opened array, but nothing happens with the v-list-group.

When you repeatedly click the v-list-group activator itself, you can see how that 'Users' string correctly appears and disappears from the array.

It looks like that v-model only works in one way (from child to parent).

**Comment by piris-jose** [comment]
Checking whether the behavior described was new or not, I confirm that this **used to work** on **Vuetify 3.6.15** with **Vue 3.4.38**.

**Comment by KaelWD** [comment]
Confirmed broken in beta 0, probably #20038

**Comment by balage1551** [comment]
Also confirm broken in 3.7.1.
And can't find workarround either. Not even reseting then setting the open array value.

**Comment by daywin94** [comment]
Did somebody find a workaround for this?

**Comment by balage1551** [comment]
> Did somebody find a workaround for this?

Yes, today I made a simple hack (until it wll be fixed in 3.7.2):

```html
<v-list v-if="!hackRefreshTrigger" ...>
   ...
```

then:

```ts
function hackTriggerFullRefresh() {
   hackRefreshTrigger.value = true
    nextTick(() => {
      hackRefreshTrigger.value = false
    })
} 
```
